src/components/ChatLog.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog =(props) => {
+  
+  // const forLike = (value) => {alert(value)}
+
+  // console.log(props)
+  // console.log(props.allMessages)
+  return props.entries.map((oneMessage) => {
+    return (
+      <ChatEntry
+        sender={oneMessage['sender']}
+        body={oneMessage['body']}
+        timeStamp={oneMessage['timeStamp']}
+        id={oneMessage['id']}
+        key={oneMessage['id']}
+        liked={oneMessage['liked']}
+        updatedMessage={props.updatedMessage}
+        // message={oneMessage}
+        // onLike={forLike}
+    ></ChatEntry>
+    )
+  })
+  }
+
+  // ChatLog.propTypes = {
+  //   entries: PropTypes.isRequired,
+  // };
+  ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(
+        PropTypes.shape({
```

Comment:
Nice use of PropTypes.shape for more detail on what data is expected for this component =]